Neos.ContentRepository.Core/Classes/Factory/CommandHooksFactoryDependencies.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\Factory;
+
+use Neos\ContentRepository\Core\Projection\ContentGraph\ContentGraphReadModelInterface;
+use Neos\ContentRepository\Core\SharedModel\ContentRepository\ContentRepositoryId;
+
+/**
+ * @internal
+ */
+final readonly class CommandHooksFactoryDependencies
```

Comment:
only the constructor must be internal i guess? its api to create those? Same for the factories?

Reply:
good point, adjusted with feb40b056ad6a6944f36142570d754a5eeccc6ec